packages/features/insights/components/RoutingFormResponsesTable.tsx

```diff
@@ -582,7 +630,7 @@ export function RoutingFormResponsesTableContent() {
             <DataTableFilters.ColumnVisibilityButton table={table} />
           </>
         }>
-        <RoutingKPICards />
+        <RoutingKPICards given={{ isAll, teamId, userId }} />
```

Comment:
We caxn provide `isAll, teamId, userId` to this component, so that it won't attempt to read them from `useFilterContext()`.